### PR TITLE
Add Tests + allow styling of `li`

### PR DESCRIPTION
**Pull request by apfelbox**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no 
| Improvement?  | yes
| BC breaks?    | no
| Deprecations? | no 
| Docs PR       | —

Finally added tests

### CODE REVIEW

File: .stylelintrc.yml [hunk 1]
```diff
@@ -93,7 +93,9 @@ rules:
     selector-pseudo-element-no-unknown: true
     selector-type-case: lower
     selector-max-empty-lines: 0
-    selector-max-type: 0
+    selector-max-type:
+        - 0
+        - ignoreTypes: ["li"]
```
Comment (keichinger) [inline .stylelintrc.yml]: Do you also want to exclude `img` and `span`? I'd guess that these are the most common element selectors that we're using. Not having to add a class everywhere might be a convenience-only feature though, so it's debatable.

**Review by keichinger — APPROVED** [review]

File: .stylelintrc.yml (hunk 1 above)
Comment (apfelbox) [reply .stylelintrc.yml]: Hmm, we can add `i`, but `span` and `img` should be avoided to be honest.

Comment (apfelbox) [reply .stylelintrc.yml]: `img`: thinkable that you want to change an `img` with a `div` with a background image, that will break horribly.

`span`: should just be avoided except in very specific cases. In this case you should add the comment disabling the lint with a comment explaining it. (that is part of the purpose of this check)

Comment (apfelbox) [reply .stylelintrc.yml]: Oh, `body` and `html` should be allowed as well.
I wouldn't enforce the "class on the `body`" rule super strict, but just add it as comment in future PRs.

**Pull request merged**